themes/mongodb/src/css/manual/mongodb-docs.css

```diff
@@ -1164,11 +1164,34 @@ div.sphinxsidebar a { color: #333; }
     font-size: 12px;
 }
 
+.version-sidebar {
+  box-sizing: border-box;
+  margin: 8px 12px 8px 22px;
+  width: calc(100% - 34px);
+}
+
+.version-sidebar .dropdown-menu {
```

Comment:
I might prefer a child combinator rather than the descendant combinator if that works here